src/js/components/message.jsx

```diff
@@ -35,16 +36,54 @@ export class MessageComponent extends Component {
         }
 
         const style = {};
-        if (isAppUser && this.props.accentColor) {
+
+        if (!this.props.mediaUrl) {
+          style.borderTopLeftRadius = style.borderTopRightRadius = style.borderBottomLeftRadius = style.borderBottomRightRadius = 16;
+          if (isAppUser && this.props.accentColor) {
             style.backgroundColor = style.borderLeftColor = `#${this.props.accentColor}`;
+          }
+          if (this.props.firstInGroup && !this.props.lastInGroup) {
+            if (isAppUser) {
+              style.borderBottomRightRadius = 2;
```

Comment:
Hmm right, instead of using inline style the widget should be using classes to allow people to override them.